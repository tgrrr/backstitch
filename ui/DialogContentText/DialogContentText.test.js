import React from 'react'
import renderer from 'react-test-renderer'
import { DialogContentText } from '.'

it('DialogContentText: default', () => {
    const component = renderer.create(
        <DialogContentText>
            <>
                DialogContentText
            </>
        </DialogContentText>
    )
  const tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
