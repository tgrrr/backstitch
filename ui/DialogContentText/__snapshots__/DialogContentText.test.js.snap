// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DialogContentText: default 1`] = `
<div
  className="sc-AxjAm StDqN"
>
  <div
    className="Dialogcontenttext "
    data-testid="Dialogcontenttext"
  >
    <p
      className="MuiTypography-root MuiDialogContentText-root MuiTypography-body1 MuiTypography-colorTextSecondary"
    >
      DialogContentText
    </p>
  </div>
</div>
`;
